### PR TITLE
Solution Submission

### PR DESCRIPTION
**Pull request by Mattaus-Isenberg**
- [ ] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401challenges/src/main/java/utilities/FizzBuzzTree.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            }
+        }
+    }
+
+    public static Tree<Object>FizzBuzzTree(Tree<Object> tree)
+    {
+        FizzBuzzTree(tree.getRoot_Node());
+        return tree;
+
+    }
```
Comment (jeffborda) [inline code401challenges/src/main/java/utilities/FizzBuzzTree.java:35]: Excellent use of a wrapper method to call your recursive method.

File: code401challenges/src/main/java/utilities/FizzBuzzTree.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            else if ((Integer) node.node_Data % 5 == 0)
+            {
+                node.node_Data = "Buzz";
+            }
+            else if ((Integer) node.node_Data % 3 == 0)
+            {
+                node.node_Data = "Fizz";
+            }
+        }
+    }
```
Comment (jeffborda) [inline code401challenges/src/main/java/utilities/FizzBuzzTree.java:28]: 👍 This will preform a post-order traversal since it recurses on the left and right before performing the fizz buzz operation.
